drivers/power/supply/regulator.c

```diff
@@ -236,6 +238,32 @@ static int _regulator_get_voltage(FAR struct regulator_dev_s *rdev)
   return ret;
 }
 
+static int _regulator_do_enable_pulldown(FAR struct regulator_dev_s *rdev)
+{
+  int ret;
```

Comment:
int ret = 0;

  if (rdev->ops->enable_pulldown)
    {
      ret = rdev->ops->enable_pulldown(rdev);
      if (ret < 0)
        {
          pwrerr("failed to enable pulldown\n");
        }
    }

  return ret;

Reply:
Yes, of course. Sloppy - will sort.

Reply:
done